src/software/thunderscope/gl/gl_widget.py

```diff
@@ -321,7 +321,7 @@ def set_camera_view(self, camera_view: CameraView) -> None:
         self.gl_view_widget.reset()
         if camera_view == CameraView.ORTHOGRAPHIC:
             self.gl_view_widget.setCameraPosition(
-                pos=pg.Vector(0, 0, 0), distance=1100, elevation=90, azimuth=-90
+                pos=pg.Vector(0, 0, 0), distance=900, elevation=90, azimuth=-90
```

Comment:
`distance=1100` looks like this for me:

![image](https://github.com/UBC-Thunderbots/Software/assets/60044853/c3345979-dbd2-432c-a8a6-24eb6f4ccb79)

while `distance=900` looks like this:

![image](https://github.com/UBC-Thunderbots/Software/assets/60044853/9ab4967b-8670-4d96-8159-3c2cecd1e469)

Maybe try getting the dimensions of the viewport and then calculate a `distance` that scales the field to the viewport?

Reply:
Sounds good, I've made those changes.